.github/workflows/testing_apps.yml

```diff
@@ -15,6 +15,22 @@ jobs:
               npm run encore dev
               npm run encore production
 
+          - name: npm (with Babel)
+            working_directory: test_apps/npm-with-babel
+            script: |
+              npm install --ci
+              npm add --save-dev ../../webpack-encore.tgz
+              npm run encore dev
+              npm run encore production
+
+          - name: npm (with external Babel configuration file)
+            working_directory: test_apps/npm-with-external-babel-config
+            script: |
+              npm install --ci
+              npm add --save-dev ../../webpack-encore.tgz
+              npm run encore dev
+              npm run encore production
+
```

Comment:
Sadly there is no way with GitHub Actions to use Yaml anchors, so we had to duplicate the script each time. 😕